### PR TITLE
Update link to form and selectors

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/SNVGe19453c4.yaml [hunk 1]
```diff
@@ -3,47 +3,47 @@ contact_form:
...
+          selector: textarea#message
           value: $MESSAGE
           required: true
     - select:
         - name: prefix
-          selector: select#dnf_class_values_communication__primary_constituent__0__prefix_
+          selector: select#Prefix
           value: $NAME_PREFIX
           required: true
           options:
```
Comment (tomdionesotes) [inline members/SNVGe19453c4.yaml]: same options?

**Comment by OdeclasV** [comment]
@tomdionesotes let me know if you have more comments/suggestions

**Comment by tomdionesotes** [comment]
@OdeclasV nope just the one question: https://github.com/NGPVAN/contact-congress/pull/887#discussion_r279864901

**Comment by OdeclasV** [comment]
gracias!

**Pull request merged**